### ISSUE TITLE
If anyone is having a problem with MO 2 alpha 4, please read.

### ISSUE THREAD

**Issue by Sir1Twist** [TanninOne/modorganizer#552]
I had failures at attempting to use Mod Organizer 2 Alpha 4. If you have a previous version of MO2, please go to your appdata\local\Mod Organizer folder, you want to move out of there the ModOrganizer.ini file. This should give you a clean start with it.

**Comment by KeltecRFB** [comment]
I just overwrote Beta 3 because I wanted to keep all of my settings and it is working as expected.

**Comment by pressure68** [comment]
Unfortunately I am just getting the splash screen and nothing else with either my previous ini or without one at all.  Win7 x64

**Comment by Sir1Twist** [comment]
I'm actually having a problem with F4SE, and FO4Edit.

**Comment by rdunlap** [comment]
Sir1Twist... What problems? I am using both with no errors at all in Alpha 4. I will see if I can recreate your issues or maybe help find them for you to fix.

**Comment by rdunlap** [comment]
I am actually becoming tempted to drop it in my skyrim install and see what happens.

**Comment by Sir1Twist** [comment]
I keep getting a component of the F4SE has stopped working. I am using the latest F4SE release, though. For Fo4Edit, I'm getting couldn't determine my documents folder. Not 100% sure what's going on with either one. I've been using both for a while.

**Comment by rdunlap** [comment]
Could you have a stray Ini file for xEdit? Or maybe a flag set in the exe for MO? OH! and will it start on its own without MO? I created the exe link in MO2A4 new and set the working directory to be Fallout 4/Data and it just works

**Comment by Sir1Twist** [comment]
The launcher and game run fine in MO 2, and F4SE runs fine outside. Just Not inside of MO2.

 I'm taking a look at what ini files I have, in mods, and around FO4. I'm thinking about just adding, if I can, the contents of the ini files to other files, and consolidate them, and even move them around, so they won't affect anything else.

**Comment by rdunlap** [comment]
not a problem on this end...
I have MO2 in a folder inside Fallout 4 and I also have FO4Edit in a folder inside Fallout 4... Now I do not have my Steam in any system generated folder like program files... I have Steam moved off to a Game Dedicated drive so I wouldn't have permissions problems. I wonder if you are being balked by permissions or by your AV since MO2 is changing in signature? do you have an exception for all of the executable components individually as well as the folder exception? I have since Alpha 2 released and my AV product started throwing fits about changed Exe's. Here is a question. Why does xEdit want to find your documents folder? for a loadorder?

Robert

**Comment by Sir1Twist** [comment]
I use Avast for my anti-virus, which I normally have no problems with. I'm trying a couple of things, that may work.Hopefully unsetting read-only from just about everything may help.

**Comment by rdunlap** [comment]
Ok, It has to be something though, if it worked so easily for me then it should for you. There is nothing exceptional about my setup. You could pause your file system protection long enough to do a test run and if it was AV related it would reveal it.

Robert

**Comment by Sir1Twist** [comment]
I reinstalled MO 2 alpha 4, and had no problems with running it. I do need to reboot my system, but I have some other stuff I am currently working on. I will reply, again at that point. For the moment, I don't seem to have the problems I had before with FO4Edit, and F4SE. I should know better after the reboot.

**Comment by Sir1Twist** [comment]
Okay, I was having a bit of a problem running Bodyslide with MO 2 alpha 4, but fixed it. It took a little searching, and a little digging into things, but here's what I did. I went to where I have it installed, right clicked on Bodyslide x64, as that's what I am using, then Properties. I then went and double-checked the run as admin was on. Then checked the actual security tab. (Key step here.) I hit advanced, and double checked there I had full control. I hit okay, then unchecked, then rechecked, under compatibility, the Run as Admin. Closed everything out, and it works. (I get no images on the preview, but it does work. I may need to tweak the settings, but it does work.) Right now, I'm patiently awaiting updates for both F4SE and FO4Edit.
